### PR TITLE
TEX-91: Added blue border to active panel preview image

### PR DESCRIPTION
**Pull request by nigelritfeld**
Added blue border to preview image. Based on active panel in the video preview.
Screenshot 2023-03-27 at 22 18 29

### CODE REVIEW

File: components/timeline/videoPlayer.tsx [hunk 1]
```diff
@@ -20,6 +21,8 @@ export const VideoPlayer: FC<VideoPlayerProps> = ({ panels }) => {
     const handleEnd = () => {
         const currentPanelIndex = panels.findIndex((panel) => panel === video);
         const nextPanelIndex = currentPanelIndex + 1;
+        setCurrent(currentPanelIndex)
+        console.log(currentPanelIndex)
```
Comment (srdegier) [inline components/timeline/videoPlayer.tsx]: Haal console.log weg

**Review by srdegier — CHANGES_REQUESTED** [review]
Door te screenshot geloof ik wel dat het werkt, verder check ff die comment over de console.log

File: components/timeline/videoPlayer.tsx (hunk 1 above)
Comment (nigelritfeld) [reply components/timeline/videoPlayer.tsx]: Done

**Review by srdegier — APPROVED** [review]
Toppie!

**Pull request merged**